CONTRIBUTING.md

```diff
@@ -20,8 +20,17 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
 We welcome icon requests. Before you submit a new issue please make sure the icon:
 
 * Has not already been requested. If you find an existing issue or pull request for the brand you're looking for then please add a reaction or comment to show your support.
-* Is of a _popular_ brand. For websites, the [Alexa rank](https://www.alexa.com/siteinfo) should be less than 500k. For anything else, popularity will be judged on a case-by-case basis.
-* Isn't related to anything that provides an illegal service (e.g. piracy, malware, threatening material, spam, etc.).
+* Is of a _popular_ brand. For websites, the [Alexa rank](https://www.alexa.com/siteinfo) should be less than 500k. For GitHub projects, the amount of "stars" should be above 5k. For anything else, popularity will be judged on a case-by-case basis.
+* Doesn't fall into one of the following categories:
+    - Anything that provides an illegal service (e.g. piracy, malware, threatening material, spam etc.)
+    - Governmental agencies, programs, departments (e.g. US Air Force, FBI) BUT we do allow international organizations and NGOs with supranational interests (e.g. WHO, Unicef, NASA, ESA, ...)
```

Comment:
Maybe we can make this a bit shorter, how about:

```suggestion
    - Governmental agencies, programs, departments. International organizations and NGOs with supranational interests (e.g. WHO and Unicef) are allowed.
```

Also removed NASA and ESA from this sentence as they're not international nor NGOs. Not sure how to mention them though? Perhaps a sublist one level deeper?